### PR TITLE
Improved caption of image

### LINKED ISSUE #2: Activity 1: Your First Caption
Using the API, I have just created a file with a random octocat for you to caption. In this activity, we will learn how to make changes to files in our GitHub repository.

- [ ] @Felicia-Little
- [ ] @Harinchakravarthi
- [ ] @Martian-Manhunter
- [ ] @andrew-scoppa
- [ ] @jcardich
- [ ] @jerbaile
- [ ] @whatlove
- [ ] @xansari
- [ ] @AngelAndrades
- [ ] @Glorr
- [ ] @JeremyJordan2
- [ ] @sdisneyCD79

### PR DESCRIPTION
**Pull request by andrew-scoppa**
Made a small font change. This PR resolves #2

### CODE REVIEW

**Comment by AngelAndrades** [comment]
Good Luck!

File: _slides/02-andrew-scoppa.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
-This is my mascot
-
-![cooler-octocat](https://octodex.github.com/images/twenty-percent-cooler-octocat.png)
-{: .center}
+---
+layout: slide
+title: "Welcome to our slide deck!"
+---
+
+_This is my favorite mascot!_
```
Comment (Glorr) [inline _slides/02-andrew-scoppa.md:6]: I thought your favorite was the other one.

Comment (andrew-scoppa) [reply _slides/02-andrew-scoppa.md:6]: You're correct!

**Comment by Martian-Manhunter** [comment]
Interesting

**Review by Glorr — APPROVED** [review]
Great job Andy.

**Pull request merged**